### ISSUE TITLE
why the validation result so low?

### ISSUE THREAD

**Issue by miaoxiaoting** [hustvl/GKT#13]
我修改了gkt_nuscenes_vehicle_kernel_7x1_setting1.yaml中BEV h的尺寸，可是训练过程中，val 指标特别低，基本不变
，不知道为什么
data:
  bev:
    # h: 400 # TODO 程序跑不通
    h: 200
    w: 200
    # h_meters: 100.0 # TODO
    h_meters: 50.0
    w_meters: 50.0
# python scripts/train.py +experiment=gkt_nuscenes_vehicle_kernel_7x1_setting1.yaml  \
# data.dataset_dir=/GKT/data/nuscenes/mini data.labels_dir=/GKT/data/nuscenes/cvt_labels_nuscenes_v2
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
# ┃            Validate metric             ┃              DataLoader 0              ┃
# ┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
# │         train/metrics/iou@0.40         │           0.9998205304145813           │
# │         train/metrics/iou@0.50         │           0.9998857975006104           │
# │ train/metrics/iou_with_occlusions@0.40 │           0.7885284423828125           │
# │ train/metrics/iou_with_occlusions@0.50 │           0.7882291674613953           │
# │                val/loss                │          0.17532268166542053           │
# │            val/loss/center             │          0.003345666453242302          │
# │            val/loss/visible            │          0.17498812079429626           │
# │          val/metrics/iou@0.40          │          0.051858142018318176          │
# │          val/metrics/iou@0.50          │           0.0490204356610775           │
# │  val/metrics/iou_with_occlusions@0.40  │          0.037373993545770645          │
# │  val/metrics/iou_with_occlusions@0.50  │          0.03504466265439987           │
# └────────────────────────────────────────┴────────────────────────────────────────┘
